.github/workflows/ci.yml

```diff
@@ -1,27 +1,14 @@
-# Each virtual machine has the same hardware resources available.
-#
-# - 2-core CPU
-# - 7 GB of RAM memory
-# - 14 GB of SSD disk space
-#
-# https://help.github.com/en/actions/automating-your-workflow-with-github-actions/virtual-environments-for-github-hosted-runners
-
 name: CI
```

Comment:
Nit: we could go with a more descriptive name than just CI. "Branch Check"?

Reply:
It runs on pushes (e.g. pushing a commit to `main` or even pushes to PR branches), so, technically it is checking a branch (but more the commits on that branch).

So, "branch check" is more specific, but I think CI is so engrained into me now it feels more right here?